storybookjs/storybook
# Ember ts

Issue: #11845 #16205

## What I did

Adds types for ember stories 

## How to test

I've included a TypeScript Story in `examples/ember-cli`

This meant I needed to add `ember-cli-typescript`. I also updated the version of `ember-cli` and friends

I'm not sure if there is anything else I should add? 👀 
Happy to make changes! 👋
